### PR TITLE
feat: setup demo doc

### PR DESCRIPTION
**Pull request by benjaminlim00**

### CODE REVIEW

File: src/components/demo-doc.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { WrappedDocument } from "@govtechsg/open-attestation";
```
Comment (Nebulis) [inline src/components/demo-doc.tsx:1]: doc => document (everywhere, file name component name, ...)

File: src/components/raw/raw-demo.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
+{
```
Comment (Nebulis) [inline src/components/raw/raw-demo.json:1]: remove

File: src/components/demo-doc.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import React from "react";
+import ROPSTEN_DEMO from "./wrapped/wrapped-demo.json";
+
+interface DemoDocProps {
+  setRawDocument: (doc: WrappedDocument) => void;
+}
+
+export const DemoDoc: React.FunctionComponent<DemoDocProps> = ({ setRawDocument }: DemoDocProps) => {
```
Comment (Nebulis) [inline src/components/demo-doc.tsx:9]: it's not used ?

Comment (benjaminlim00) [reply src/components/demo-doc.tsx:9]: DemoDoc is being used in home.tsx

File: src/components/demo-doc.tsx (hunk 1 above)
Comment (benjaminlim00) [reply src/components/demo-doc.tsx]: done

File: src/components/raw/raw-demo.json (hunk 2 above)
Comment (benjaminlim00) [reply src/components/raw/raw-demo.json]: done

File: src/components/demo-doc.tsx (hunk 3 above)
Comment (Nebulis) [reply src/components/demo-doc.tsx]: https://deploy-preview-15--verify-gov-sg.netlify.app/

where ?

Comment (Nebulis) [reply src/components/demo-doc.tsx]: image

Comment (benjaminlim00) [reply src/components/demo-doc.tsx]: can you try to clear cache and deploy on netlify? because it is running fine on local but now showing up on netlify

Comment (benjaminlim00) [reply src/components/demo-doc.tsx]: I fixed this issue

File: src/pages/home.tsx [hunk 4]
```diff
@@ -31,7 +32,10 @@ export const HomePage: React.FunctionComponent = () => {
 
   return (
     <div className="bg-gray-100">
-      <DropZone setRawDocument={(doc) => setRawDocument(doc)} />
+      <div className="flex">
+        <DemoDocument setRawDocument={(document) => setRawDocument(document)} />
+        <DropZone setRawDocument={(document) => setRawDocument(document)} />
```
Comment (Nebulis) [inline src/pages/home.tsx:37]: I never noticed that ... setRawDocument is coupling to what the user intend to do, it should be  named sth like => onFileLoaded, onDocumentLoaded, onDocumentDropped,

**Pull request merged**